Modules/HypnokinkModule/Scenes/HK_VionRepeatConversation.gd

```diff
@@ -0,0 +1,274 @@
+	if(state == "ask_chastity"):
+		saynn("[say=pc]So I take it Dr. Quinn's up to her usual shenanigans?[/say]")
+
+		saynn("You gesture in the direction of his crotch. He chuckles.")
+
+		saynn("[say=Vion]Hah, I wish. Then I might be able to [i]do something[/i] about it. No, this is one last gift from my ex. One of the most premium cages on the market, actually, a DeLoxe S3-Karat model. I don't have access to a device to input the password on, and anyway the one I know will have expired by now![/say]")
+		
+		saynn("He huffs.")
+		
+		saynn("[say=Vion]Nothing short of of a sub-molecular disassembler is getting this off without castrating me in the process. And for some inexplicable reason, cutting tools are in very short supply in a high-security prison facility.[/say]")
+
+		saynn("[say=Vion]Funny how the usually ever so diligent guards seemed to overlook it during the intake process, isn't it?[/say]")
+
+		saynn("He idly kicks at the chair, making it rattle across the floor. You suspect {Vion.name} does [i]not[/i] find the joke very funny.")
+
+		setFlag("HypnokinkModule.VionTopicKnown_Chastity", true)
+		addButton("Back", "", "questions")
+	if(state == "discount_rush_intro"):
+		saynn("[say=pc]Your skills don't exactly come cheap.[/say]")
+
+		saynn("[say=Vion]Yes, I'm aware. The truth is, I would prefer to price my sessions more conservatively, but I'm rather pressed for credits.[/say]")
+
+		saynn("[say=pc]Why? It isn't like you have to pay rent in here, at least.[/say]")
+
+		saynn("[say=Vion]Yes, well, there's a certain individual that has made it his business to make certain demands of me. So far I've been able to pay him off instead, but the asking price's been going up over time. I do have my suspicions about his motives, but given that it's [i]literally[/i] my ass on the line, I'd rather not wager on a hunch.[/say]")
+
+		saynn("[say=pc]What's he like? Maybe I can deal with him for you.[/say]")
+
+		saynn("[say=Vion]Are you sure? Not the kind of person you want to make angry. He's... well... he's big.[/say]")
+
+		saynn("Vion's eyes unfocus for a moment as he stares at nothing, presumably experiencing a vivid memory of unusual size.")
+
+		saynn("[say=pc]Big?[/say]")
+
+		saynn("[say=Vion]Trust me, when you see him you'll agree its an accurate descriptor. Anyway, he's a hybrid of equine and dragon. Neon blue hair in a mohawk, dark furred. High sec inmate, goes by the name of Rush. When he's not wandering around and harrasing me, you can usually find him in the gym or the high security cellblock.[/say]")
+
+		saynn("[say=pc]Okay, I'll see what I can do.[/say]")
+
+		saynn("[say=Vion]Just... be careful. I don't want a broken spine on my conscience. Maybe consider pursuing, well, alternate methods of diplomacy?[/say]")
+
+		addButton("Back", "", "")
+		
+	if(state == "discount_onthehouse"):
+		saynn("[say=pc]How much will this cost me?[/say]")
+		
+		saynn("[say=Vion]For you? Sessions are on the house.[/say]")
```

Comment:
It kinda sounds like they will be free forever. Maybe adding a simple 'for now' at the end would make it less confusing. Sorry that I nitpick text x3

Reply:
Split it between a temporary and permanent comment.